### PR TITLE
[HIVEMALL-116][SPARK][DOC] Add docs for SQL cases in hivemall-spark

### PR DESCRIPTION
**Pull request by maropu**
## What changes were proposed in this pull request?
This pr added docs for SQL cases in `hivemall-spark`.

## What type of PR is it?
Documentation

## What is the Jira issue?
https://issues.apache.org/jira/browse/HIVEMALL-116

## How was this patch tested?
N/A

### CODE REVIEW

File: docs/gitbook/spark/regression/e2006_sql.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+```scala
+scala> :paste
+spark.read.format("libsvm").load("E2006.train.bz2")
+  .select($"label", to_hivemall_features($"features").as("features"))
+  .createOrReplaceTempView("rawTrainTable")
+
+// `label` must be [0.0, 1.0]
+sql("""
+  CREATE OR REPLACE TEMPORARY VIEW trainTable AS
+    SELECT rescale(label, -7.899578, -0.51940954) AS label, features
```
Comment (myui) [inline docs/gitbook/spark/regression/e2006_sql.md]: `-7.899578, -0.51940954` can better be expressed as `${min}, ${max}` as follows: 

```scala
val rdd = sql("select min(label) as min, max(label) as max from rawTrainTable")
val row = rdd.first
val min = row.get("min")
val max = row.get("max")
sql("""  
  ...
```

File: docs/gitbook/spark/regression/e2006_sql.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+scala> :paste
+spark.read.format("libsvm").load("E2006.test.bz2")
+  .select($"label", to_hivemall_features($"features").as("features"))
+  .createOrReplaceTempView("rawTestTable")
+
+sql("""
+  CREATE OR REPLACE TEMPORARY VIEW testTable AS
+    SELECT
+        rowid() AS rowid,
+        rescale(label, -7.899578, -0.51940954) AS target,
```
Comment (myui) [inline docs/gitbook/spark/regression/e2006_sql.md]: same as the above.

File: docs/gitbook/spark/regression/e2006_sql.md (hunk 1 above)
Comment (maropu) [inline docs/gitbook/spark/regression/e2006_sql.md]: FIxed

**Comment by myui** [comment]
LGTM. @maropu Could you merge this pr to master?

**Comment by maropu** [comment]
Merged to master

**Comment by myui** [comment]
Thanks!!

**Pull request closed without merging**